### PR TITLE
Move materialized view metadata info

### PR DESCRIPTION
**Pull request by mosabua**

### CODE REVIEW

**Comment by mosabua** [comment]
Dang .. I guess we missed the 362 release.. next time ..

**Comment by mosabua** [comment]
Ping @hashhar or @electrum  ;-)

**Review by hashhar — APPROVED** [review]
LGTM.

It's not ordered alphabetically anymore but I think it makes more sense since the important ones are documented first.

File: docs/src/main/sphinx/connector/system.rst [hunk 1]
```diff
@@ -50,6 +50,46 @@ that can be set when creating a new schema.
...
+.. _system_metadata_materialized_views:
+
+``metadata.materialized_views``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The materialized views table contains the following information about all
+:ref:`materialized views <sql-materialized-views-management>`:
+
+.. list-table:: Metadata for materialized views
+  :widths: 50, 50
```
Comment (ebyhr) [inline docs/src/main/sphinx/connector/system.rst:62]: We may want to change widths to `30, 70` in follow-up PR.
Screen Shot 2021-10-02 at 11 16 23

Screen Shot 2021-10-02 at 11 17 55

**Review by ebyhr — APPROVED** [review]

**Comment by ebyhr** [comment]
Merged, thanks!

File: docs/src/main/sphinx/connector/system.rst (hunk 1 above)
Comment (mosabua) [reply docs/src/main/sphinx/connector/system.rst:62]: Sure ... did you want to send that PR and get it merged before 363 @ebyhr ?

**Pull request merged**